api-reference/10 UI Components/dxDataGrid/1 Configuration/summary/calculateCustomSummary.md

```diff
@@ -25,10 +25,10 @@ Indicates the stage of the summary item calculation; equals *"start"*, *"calcula
 The resulting summary item's value.
 
 ##### field(options.value): any
-If the custom summary item is calculated by a [column](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/summary/totalItems/column.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column'), this field contains the value from a cell of this column. Otherwise, it contains a whole object from the data source.
+If the **totalItems**.[column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) property is set, this field contains the value from this column. If **totalItems**.[showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#showInColumn) is used instead, this field contains all values from a row.
```

Comment:
As I see, the information was already there, yet readers still don't notice it. Let's change the description more drastically then:

```suggestion
A column value used in calculation. If you need an entire data object in this field, do not specify the summary item's [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) property.
```